tests/EcbCurrencyRateProviderTest.php

```diff
@@ -87,7 +88,8 @@ function (ICurrency $currency, ICurrencyRateProvider $provider, \DateTime $date,
 
         $this->currencyRateProvider = new EcbCurrencyRateProvider(
             $currencyRateManager,
-            $currencyManager
+            $currencyManager,
+            new XMLLoader()
```

Comment:
The same as here https://github.com/redco/redcode-currency-rate/pull/12/files#r43361903
Use only mocking for XMLLoader in provider tests